### DISCUSSION TITLE
在powershell使用xmake显示找不到VS的版本而在git bash中则直接MINGW64编译成功是为什么

### DISCUSSION THREAD

**Discussion by Liber1917** [xmake-io/xmake#4304] [Q&A]
PS C:\Users\Desktop\hello> xmake
checking for the architecture ... x64
checking for the Microsoft Visual Studio (x64) version ... no
please run:
    - xmake config --vs=xxx [--vs_toolset=xxx]
or  - xmake global --vs=xxx

XX@xxxxxxxx MINGW64 ~/Desktop/hello (master)
$ xmake
checking for the architecture ... x86_64
[ 50%]: compiling.release src\main.c
[ 75%]: linking.release hello.exe
[100%]: build ok!

**Comment by waruqi** [comment]
cmd/ps 默认走 vs ， mingw64/msys2/git bash 下 默认走 mingw 平台。。

msvc 找不到，可以确认下 vs 有没有正常安装

**Comment by Liber1917** [comment]
我刚刚试了下，可以xmake f -p mingw直接使用mingw编译(不想下vs)，感谢
